src/main/scala/mesosphere/marathon/api/RestResource.scala

```diff
@@ -64,16 +64,15 @@ trait RestResource {
     * See [[assumeValid]], which is preferred to this.
     *
     * @param t object to validate
-    * @param description optional description which might be injected into the failure message
     * @param fn function to execute after successful validation
     * @param validator validator to use
     * @tparam T type of object
     * @return returns a 422 response if there is a failure due to validation. Executes fn function if successful.
     */
-  protected def withValid[T](t: T, description: Option[String] = None)(fn: T => Response)(implicit validator: Validator[T]): Response = {
+  protected def withValid[T](t: T)(fn: T => Response)(implicit validator: Validator[T]): Response = {
```

Comment:
Where is the description coming from now?

Reply:
It isn't needed